AndroidManifest.xml

```diff
@@ -143,7 +145,7 @@
             android:name=".authentication.AuthenticatorActivity"
             android:exported="true"
             android:launchMode="singleTask"
-            android:theme="@style/Theme.ownCloud.noActionBar" >
+            android:theme="@style/Theme.ownCloud.Toolbar" >
```

Comment:
That's not what I see in the style. 

In v21/styles.xml the style includes ```<item name="windowNoTitle">true</item>``` and ```<item name="windowActionBar">false</item>```; and in "general" styles.xml defines same values, and also extends them ```Theme.AppCompat.Light.NoActionBar```. 

Then, since ```AuthenticatorActivity``` does **not** extend ```ToolbarActivity```, there is no toolbar in it, what is right. And there is no styling info for toolbar or action bar or title, what is right too. So, perfect for this Activity :)